packages/core/helper-plugin/src/components/tests/ConfirmDialog.test.tsx

```diff
@@ -83,4 +81,34 @@ describe('ConfirmDialog', () => {
       expect(screen.getByText('Are you sure you want to unpublish it?')).toBeInTheDocument();
     });
   });
+
+  it('renders Body component part and matches text', async () => {
+    render(
+      <ThemeProvider theme={lightTheme}>
+        <IntlProvider locale="en" messages={{}} defaultLocale="en" textComponent="span">
+          <ConfirmDialog.Body>Are you sure you want to unpublish it? </ConfirmDialog.Body>
+        </IntlProvider>
+      </ThemeProvider>
+    );
+    await waitFor(() => {
+      expect(screen.getByText('Are you sure you want to unpublish it?')).toBeInTheDocument();
+    });
+  });
+
+  it('renders Root component part and matches text', async () => {
```

Comment:
I don't think its necessary to test these components separately, instead wrap `<ConfirmDialog>` with 
`<Root><Body><ConfirmDialog>{...}</ConfirmDialog/><Body/><Root/>`